Find-and-Replace/Replace-line-break-with-paragraph-mark/.NET-Standard/Replace-line-break-with-paragraph-mark/Program.cs

```diff
@@ -0,0 +1,147 @@
+                        document.Save(outputFileStream, FormatType.Docx);
+                    }
+                }
+            }
+        }
+        /// <summary>
+        /// Replace the line break with paragraph mark in the Word document.
+        /// </summary>
+        private static void ReplaceLineBreakWithPara(WordDocument document)
+        {
+            foreach (WSection section in document.Sections)
+            {
+                //Access the body section of Word document.
+                WTextBody sectionBody = section.Body;
+                IterateTextBody(sectionBody);
+                WHeadersFooters headersFooters = section.HeadersFooters;
+                //Iterate through the TextBody of OddHeader and OddFooter.
+                IterateTextBody(headersFooters.OddHeader);
+                IterateTextBody(headersFooters.OddFooter);
+            }
+        }
+
+        /// <summary>
+        /// Iterate textbody child elements.
+        /// </summary>
+        private static void IterateTextBody(WTextBody textBody)
+        {
+            //Iterate the child items of text body.
+            for (int i = 0; i < textBody.ChildEntities.Count; i++)
+            { 
+                //Check the text body items.
+                IEntity bodyItemEntity = textBody.ChildEntities[i];
+                //Check the entity type.
+                switch (bodyItemEntity.EntityType)
+                {
+                    case EntityType.Paragraph:
+                        WParagraph paragraph = bodyItemEntity as WParagraph;
+                        //Iterate through the paragraph.
+                        IterateParagraph(paragraph.Items);
+                        break;
+                    case EntityType.Table:
+                        //Iterate through table.
+                        IterateTable(bodyItemEntity as WTable);
+                        break;
+                    case EntityType.BlockContentControl:
+                        BlockContentControl blockContentControl = bodyItemEntity as BlockContentControl;
+                        //Iterate to the body items of Block Content Control.
+                        IterateTextBody(blockContentControl.TextBody);
+                        break;
+                }
+            }
+        }
+        /// <summary>
+        /// Iterate table child elements.
+        /// </summary>
+        private static void IterateTable(WTable table)
+        {
+            //Iterate the row collection in a table.
+            foreach (WTableRow row in table.Rows)
+            {
+                //Iterate the cell collection in a table row.
+                foreach (WTableCell cell in row.Cells)
+                {
+                    IterateTextBody(cell);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Iterate paragraph child elements.
+        /// </summary>
+        private static void IterateParagraph(ParagraphItemCollection paraItems)
```

Comment:
Can you change method name as "IterateParagraphItems" to convey proper work of method